CHANGELOG.md

```diff
@@ -1,22 +1,13 @@
 # Changelog
 
-## [v1.0.1](https://github.com/seegno/range-specifier-parser/tree/v1.0.1) (2016-05-18)
-[Full Changelog](https://github.com/seegno/range-specifier-parser/compare/v1.0.0...v1.0.1)
+## [1.0.2](https://github.com/uphold/range-specifier-parser/releases/tag/v1.0.2) (2019-11-11)
+- Add MIT license [\#6](https://github.com/uphold/range-specifier-parser/pull/6) ([cristianooliveira](https://github.com/cristianooliveira))
+- Improve readme and add release instructions [\#5](https://github.com/uphold/range-specifier-parser/pull/5) ([nunofgs](https://github.com/nunofgs))
+- Fix parsing of falsy range [\#4](https://github.com/uphold/range-specifier-parser/pull/4) ([nunofgs](https://github.com/nunofgs))
 
-**Merged pull requests:**
+## [v1.0.0](https://github.com/uphold/range-specifier-parser/releases/tag/v1.0.0) (2016-05-18)
```

Comment:
This seems to be incorrect because of the tags and pull requests.

Reply:
Indeed. The releases for v1.0.1 and v1.0.2 have never been released on GitHub, so this generator isn't counting with them.

Reply:
Maybe we should do that releases on GitHub?